source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -65,7 +72,7 @@ deployment and your security needs.
 
       sudo systemctl restart systemd-sysctl
 
-#. Enable kubelet service:
+#. Enable the kubelet service:
 
    .. code-block:: bash
 
```

Comment:
Add:

Enable and start the Docker service:

.. code-block:: bash

sudo systemctl enable docker
sudo systemctl start docker

Reply:
@bktan8  The "Configure" section here is generic for both Docker/CRI-O methods. Are the Docker steps required here or should we leave them in the Docker-specific section below?

Reply:
Docker is need for both.

Reply:
Why Docker is needed here? 

> sudo systemctl start docker

not needed since ```kubeadm``` starts docker if needed

Reply:
@mythi : In my experience, I had to start Docker otherwise, it didn't work.  See here:

clear@clr-k8s-master ~ $ sudo -E kubeadm config images pull
failed to pull image "k8s.gcr.io/kube-apiserver-amd64:v1.11.3": exit status 1

clear@clr-k8s-master ~ $ sudo systemctl start docker

clear@clr-k8s-master ~ $ sudo -E kubeadm config images pull
[config/images] Pulled k8s.gcr.io/kube-apiserver-amd64:v1.11.3
[config/images] Pulled k8s.gcr.io/kube-controller-manager-amd64:v1.11.3
[config/images] Pulled k8s.gcr.io/kube-scheduler-amd64:v1.11.3
[config/images] Pulled k8s.gcr.io/kube-proxy-amd64:v1.11.3
[config/images] Pulled k8s.gcr.io/pause:3.1
[config/images] Pulled k8s.gcr.io/etcd-amd64:3.2.18
[config/images] Pulled k8s.gcr.io/coredns:1.1.3

Reply:
Looks like a bug in ```kubeadm``` (the container runtime isn't started on ```images pull``` path). I still don't think we should mention docker in this section (by default kubernetes uses CRI-O and if docker is added here we end up having docker daemon running but it's not used)

Reply:
the docker steps should not be added. while the failure mentioned with the given command can occur 1) we don't explain using it, 2) the same is part of ```kubeadm init```